operator/pkg/controllers/etcd/pod.go

```diff
@@ -103,6 +103,9 @@ func podSpecFor(controlPlane *v1alpha1.ControlPlane) *v1.PodSpec {
 				"--trusted-ca-file=/etc/kubernetes/pki/ca.crt",
 				"--logger=zap",
 				"--quota-backend-bytes=8589934592",
+				"--max-request-bytes=1572864",
```

Comment:
is this the max that we could today ?

Reply:
This could be increased but we're using the same as the upstream default, https://github.com/etcd-io/etcd/blob/bfb9aa42059d84ea60a4cddd75d7f3e94b27935c/server/embed/config.go#L61. It's possible to use a larger value but we'd get a warning, https://github.com/etcd-io/etcd/blob/c83b1ad9bad637fd02c6e17010755c9b35860bd7/server/etcdserver/bootstrap.go#L56